arbstate/inbox.go

```diff
@@ -104,6 +104,9 @@ func parseSequencerMessage(ctx context.Context, batchNum uint64, batchBlockHash
 		}
 
 		if !foundDA {
+			if daprovider.IsEigenDAMessageHeaderByte(payload[0]) {
+				log.Error("No EigenDA Reader configured, but sequencer message found with EigenDA header")
```

Comment:
could this crash the binary instead, or are there reasons for this configuration to be allowed to run?

Reply:
hmm but what about the case where EigenDA is in-fact disabled but an eigenda message is posted to the inbox by malicious sequencer? if we crash binary then sequencer could bork chain iiuc